pom.xml

```diff
@@ -1339,6 +1340,13 @@
                 <artifactId>gson</artifactId>
                 <version>${gson.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>com.zaxxer</groupId>
+                <artifactId>HikariCP</artifactId>
+                <version>${hikaricp.version}</version>
```

Comment:
how about adding `test` scope here?

Reply:
We don't manage `<scope>` in `<dependencyManagement>` block